gremlin/templates/_helpers.tpl

```diff
@@ -39,7 +39,7 @@ In later versions of this chart, we will remove the use of the fallback value of
 {{- if .Values.gremlin.secret.managed -}}
 {{- default .Values.gremlin.secret.name "gremlin-secret" -}}
 {{- else -}}
-{{- .Values.gremlin.secret.name "gremlin-team-cert" -}}
+{{- default .Values.gremlin.secret.name "gremlin-team-cert" -}}
```

Comment:
For *some* reason this got deleted from the merge from https://github.com/gremlin/helm/commit/7b47554ef15ac7ea7fb8ea13d611804461142aaa

No idea

Reply:
This was my bad. I also swapped the argument order to `default` which caused more issues. We'll want to put some helm tests in place here to catch mistakes like this.